src/main/features/claim/routes/start.ts

```diff
@@ -8,7 +8,7 @@ import { DraftCertificateOfService } from 'drafts/models/draftCertificateOfServi
 export default express.Router()
 
   .get(Paths.startPage.uri, (req: express.Request, res: express.Response) => {
-    res.render(Paths.startPage.associatedView)
+    res.redirect('https://www.gov.uk/government/publications/myhmcts-how-to-make-a-damages-claim-online/issue-a-claim-for-damages-with-myhmcts') // redirect LRPB issue claim page to issue damages claims page
   })
 
   .post(Paths.startPage.uri, async (req: express.Request, res: express.Response, next: express.NextFunction): Promise<void> => {
```

Comment:
I believe it would be beneficial to delete this route as I believe it may hinder the user from making a claim.

Reply:
The user should not be able to make a claim from the LRPB. I've also updated the page to disallow HTTP Posts to the uri.